modules/nixvim/nixvim.nix

```diff
@@ -13,16 +13,33 @@
     };
   };
 
-  imports = [
-    # Added: 2024-08-06
-    (lib.mkRenamedOptionModule
-      [ "stylix" "targets" "nixvim" "transparent_bg" "main" ]
-      [ "stylix" "targets" "nixvim" "transparentBackground" "main" ])
+  imports = let
+    mkRenamedOptionModuleWith = let
+      scope = [ "stylix" "targets" "nixvim" ];
+    in
+      {
+        from,
+        to,
+      }:
+        lib.mkRenamedOptionModuleWith {
+          from = scope ++ from;
+          sinceRelease = 2411;
+          to  = scope ++ to;
+        };
+  in [
```

Comment:
Although this reduces repetition, I think it would be shorter and less confusing to just repeat the full arguments for each call to `mkRenamedOptionModuleWith`.

Reply:
> Although this reduces repetition, I think it would be shorter and less confusing to just repeat the full arguments for each call to `mkRenamedOptionModuleWith`.

Should the entire commit https://github.com/danth/stylix/pull/514/commits/03222f979eacc854e650966e5c77bd115d291973 ("treewide: generate deprecation warnings") be reverted?

Reply:
Yes, that's the change I'm referring to, so the final version should end up looking something like

```nix
[
  (lib.mkRenamedOptionModuleWith {
    from = [ ... ];
    to = [ ... ];
    sinceRelease = 2411;
  })
  (lib.mkRenamedOptionModuleWith {
    from = [ ... ];
    to = [ ... ];
    sinceRelease = 2411;
  })
]
```